source/auth/tests/mongodb-oidc.rst

```diff
@@ -160,6 +220,9 @@ of listening for SASL commands.
       }
     }
 
+Note: the driver MUST either use a unique ``appName`` or explicitly
+remove the ``failCommand`` after the test to prevent leakage.
+
```

Comment:
if we use speculative auth, the below steps regarding saslStart will be different